v2/samples/appconfiguration/v20240601/refs/v1api20200601_resourcegroup.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: resources.azure.com/v1api20200601
+kind: ResourceGroup
+metadata:
+  name: aso-sample-rg
```

Comment:
I don't think we need/want this in `refs`, do we?

The samples make it automatically so we always just assume existence in other samples AFAIK